backend/codesirius/problems/views/sample_test.py

```
@@ -0,0 +1,72 @@
+        try:
+            problem = Problem.objects.prefetch_related("sample_tests").get(
+                pk=problem_pk
+            )
+            sample_test = problem.sample_tests.get(pk=sample_test_pk)
+            sample_test.delete()
+            return CodesiriusAPIResponse(
+                status_code=status.HTTP_204_NO_CONTENT,
+                message="Sample test deleted",
+            )
+        except Problem.DoesNotExist:
+            raise ValidationError("Problem not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)